hedera-mirror-rest/__tests__/specs/contracts-result-details-01-timestamp.spec.json

```
@@ -17,6 +17,27 @@
         "payer_account_id": 8001
       }
     ],
+    "contractlogs": [
+      {
+        "consensus_timestamp": 167654000123456,
+        "contract_id": 5001,
+        "index": 0,
+        "root_contract_id": 5001
+      },
+      {
+        "consensus_timestamp": 167654000123456,
+        "contract_id": 5002,
```

```suggestion
        "contract_id": 5001,
```

I don't think it's possible under `/api/v1/5001/results/167654.000123456`, we have a log entry with a contract_id of `5002`

---

I believe we can.  If we want to show all logs for a given contract execution, that should include logs from any contracts called by that contract, so in this case contract 5001 would call contract 5002 and we would see the logs for both.  The rest of the details would be pertaining exclusively to 5001, but we want to include the full trace of the logs.

---

Thanks for the clarification. Now I see the root contract id is 5001.